### PR TITLE
Added CVE-2009-3548 Information

### PR DESCRIPTION
**Pull request by ethandellaposta**

### CODE REVIEW

File: cves/CVE-2009-3548.yml [hunk 1]
```diff
@@ -74,8 +81,8 @@ incomplete_fix_instructions: |
   Please list any fix commits for this vulnerability that had to be corrected
   at a later date.
 incomplete_fixes:
-  - commit:
-    note:
+  - commit: 
+    note: 
   - commit:
     note:
```
Comment (mesternefeld) [inline cves/CVE-2009-3548.yml:88]: I kinda like this vulnerability. I'll give it 2 upvotes

File: cves/CVE-2009-3548.yml [hunk 2]
```diff
@@ -116,9 +123,12 @@ discovered:
     If there is no evidence as to how this vulnerability was found, then you
     may leave the entries blank except for "answer", BUT please write down
     where you looked in "answer".
-  answer:
-  date:
-  automated:
+  answer: |
+    This issue was reported directly to the tomcat users public mailing
+    list by David Horheim. Link - 
+    https://markmail.org/thread/wfu4nff5chvkb6xp#query:+page:1+mid:bva3jqihjxscatev+state:results
```
Comment (mesternefeld) [inline cves/CVE-2009-3548.yml]: Does it say specifically how this vulnerability was found?

File: cves/CVE-2009-3548.yml [hunk 3]
```diff
@@ -43,29 +43,36 @@ description_instructions: |
...
+   - commit: cb8855a65c580e332681f61a7d6edfa509588bf0
+     note: Github revision, found through tomcat by following date of fix
 vccs:
-  - commit:
-    note:
+  - commit: 7f6d0018bbb36b63f4651e453be61ab298993297
+    note: |
+      rmaucher added entire file (res/tomcat.nsi) during commit with message 
+      "More release scripts work" on July 12 2006. So it appears this vulnerability was 
+      introduced upon file inception.
```
Comment (Mirldog) [inline cves/CVE-2009-3548.yml]: Could use some context on who rmaucher is as this person is not previously mentioned

**Review by Mirldog — COMMENTED** [review]
Solid work, only minor changes needed

**Comment by ghost** [comment]
2 upvotes - pretty bad oversight

File: cves/CVE-2009-3548.yml (hunk 1 above)
Comment (ethandellaposta) [reply cves/CVE-2009-3548.yml:88]: Thank you!

File: cves/CVE-2009-3548.yml (hunk 2 above)
Comment (ethandellaposta) [reply cves/CVE-2009-3548.yml]: He mentioned that while downloading exe version of the installer (specifically not zip or tar.gz) it enables the manager application - which leads to the default blank password.

File: cves/CVE-2009-3548.yml (hunk 3 above)
Comment (ethandellaposta) [reply cves/CVE-2009-3548.yml]: I will update with some background info on who this person is within the organization.

**Pull request merged**